mkosi

```diff
@@ -2107,59 +2104,33 @@ def install_debian_or_ubuntu(args: CommandLineArguments,
         cmdline.append("--no-check-valid-until")
 
     cmdline += [args.release, root, mirror]
-
-    if not do_run_build_script and args.bootable and args.output_format == OutputFormat.gpt_btrfs:
-        cmdline[4] += ",btrfs-progs"
-
     run(cmdline, check=True)
 
-    # Debootstrap is not smart enough to deal correctly with alternative dependencies
-    # Installing libpam-systemd via debootstrap results in systemd-shim being installed
-    # Therefore, prefer to install via apt from inside the container
-    extra_packages = ['dbus', 'libpam-systemd', 'binutils']
-
-    # Also install extra packages via the secondary APT run, because it is smarter and
-    # can deal better with any conflicts
-    extra_packages.extend(args.packages)
+    # Install extra packages via the secondary APT run, because it is smarter and can deal better with any
+    # conflicts.
+    extra_packages = {"systemd", "systemd-sysv"}
+    extra_packages.update(args.packages)
 
     if do_run_build_script:
-        extra_packages.extend(args.build_packages)
-
-    # Work around debian bug #835628
-    os.makedirs(os.path.join(root, "etc/dracut.conf.d"), exist_ok=True)
-    with open(os.path.join(root, "etc/dracut.conf.d/99-generic.conf"), "w") as f:
-        f.write("hostonly=no")
+        extra_packages.update(args.build_packages)
 
     if not do_run_build_script and args.bootable:
-        extra_packages += ["dracut"]
+        if args.with_unified_kernel_images:
+            extra_packages.add("dracut")
+            extra_packages.add("binutils")
+        else:
+            extra_packages.add("initramfs-tools")
+
         if args.distribution == Distribution.ubuntu:
-            extra_packages += ["linux-generic"]
+            extra_packages.add("linux-generic")
         else:
-            extra_packages += ["linux-image-amd64"]
+            extra_packages.add("linux-image-amd64")
 
         if args.bios_partno:
-            extra_packages += ["grub-pc"]
-
-    # Debian policy is to start daemons by default.
-    # The policy-rc.d script can be used choose which ones to start
-    # Let's install one that denies all daemon startups
-    # See https://people.debian.org/~hmh/invokerc.d-policyrc.d-specification.txt
-    # Note: despite writing in /usr/sbin, this file is not shipped by the OS
-    # and instead should be managed by the admin.
-    policyrcd = os.path.join(root, "usr/sbin/policy-rc.d")
-    with open(policyrcd, "w") as f:
-        f.write("#!/bin/sh\n")
-        f.write("exit 101")
-    os.chmod(policyrcd, 0o755)
```

Comment:
I don't think we should change that. This has been in the codebase for two years with nobody complaining and changing it now would be a backwards incompatible change, since people building Debian/Ubuntu images would need to disable all the services that are now started.